app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb

```diff
@@ -6,11 +6,11 @@
 
 <div class="content">
   <section class="col col-720">
-    <p>Bursaries are tax-free payments you receive if you're training to teach certain subjects. If you're eligible for a bursary, you'll get it automatically.</p>
-    <p>Scholarships are awarded by organisations with a passion for their subject area. They combine tax-free payments with extra benefits like subject-specific support, mentoring and networking opportunities. If you're eligible, you can apply directly to the scholarship provider.</p>
+    <p>Postgraduate bursaries are tax-free payments you receive if you're training to teach certain subjects. You do not need to apply for a bursary, you'll automatically get one if you're eligible.</p>
+    <p>Postgraduage scholarships are awarded by organisations with a passion for their subject area. They combine tax-free payments with extra benefits like subject-specific support, mentoring and networking opportunities. If you're eligible, you can apply directly to the scholarship provider.</p>
     <p>You cannot receive both a bursary and a scholarship.</p>
     <p>If you receive either a bursary or a scholarship, you do not need to pay it back.</p>
-    <p>Bursaries and scholarships are not available for:</p>
+    <p>Postgraduate bursaries and scholarships are not available for:</p>
```

Comment:
In the context of the page, I found this a bit confusing as in the sentences above we just say bursaries and scholarships